components/Shared/TopNav.tsx

```diff
@@ -0,0 +1,73 @@
+import Logo from "@/public/assets/icons/dashboard/Logo.svg";
+import {
+	ArrowDownIcon,
+	MenuIcon,
+	NotificationIcon,
+	PlusButtonIcon,
+} from "@/public/assets/icons/dashboard/icons";
+import userIcon from "@/public/assets/icons/dashboard/userIcon.svg";
+import Image from "next/image";
+import Link from "next/link";
+import { Button } from "../ui";
+
+type TopNavProps = {
+	toggleSidebar: () => void;
+};
+
+export default function TopNav({ toggleSidebar }: TopNavProps) {
+	return (
+		<div className="sticky top-0 flex items-center justify-between border-b border-gray-400 p-7  md:border-gray-300 md:p-4 md:pr-10">
```

Comment:
this should be a header element because this is a component that will be ontop of the page as a header